src/core/lib/iomgr/exec_ctx.cc

```diff
@@ -109,6 +109,11 @@ grpc_closure_scheduler* grpc_schedule_on_exec_ctx = &exec_ctx_scheduler;
 namespace grpc_core {
 GPR_TLS_CLASS_DEF(ExecCtx::exec_ctx_);
 
+void ExecCtx::TestOnlyGlobalInit(gpr_timespec new_val) {
```

Comment:
Add a comment with a warning that this function is used for testing purposes only

Reply:
Done.